src/core/config.c

```diff
@@ -16,6 +16,7 @@ static const char *ini_keys[] = {
     "gameplay_fix_100y_ghosts",
     "screen_display_scale",
     "screen_cursor_scale",
+    "screen_display_speedrun_infos",
```

Comment:
The `screen_` prefix is for scaling-related properties. `ui_` is for change to the interface. This change is a UI change, so please rename it to `ui_show_speedrun_info` to be consistent with the other settings. Also note I dropped the "s": "info" is short for "information" which has no plural.